pkgs/development/python-modules/EasyOCR/default.nix

```
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hdf5
+, numpy
+, opencv3
+, pillow
+, pyaml
+, pyclipper
+, python-bidi
+, pytorch
+, scikitimage
+, scipy
+, shapely
+, torchvision
+}:
+
+buildPythonPackage rec {
+  pname = "EasyOCR";
+  version = "1.6.2";
+
+  src = fetchFromGitHub {
+    owner = "JaidedAI";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-f+JBSnFMRvVlhRRiL1rJb7a0CNjZPuh6r8r3K1meQCk=";
+  };
+
+  postPatch = ''
+    substituteInPlace requirements.txt \
+      --replace "opencv-python-headless<=4.5.4.60" ""
+    substituteInPlace requirements.txt \
+      --replace "ninja" ""
+  '';
+
+  propagatedBuildInputs = [
+    scikitimage
+    hdf5
+    python-bidi
+    numpy
+    opencv3
+    torchvision
+    pillow
+    pyaml
+    pyclipper
+    pytorch
+    scipy
+    shapely
+  ];
+
+  doCheck = false; # fails as cannot find correct onnx dep
```

```suggestion
  checkInputs = [
    onnx
  ];
```
onnx is available in nixpkgs.

---

Oh, available but broken on master.

---

I fixed onnx (NixOS/nixpkgs#197194), so could you include it?

---

Done.